cablelabs/lpwanserver
# Issue 333

#### What does this PR do?
- Update the OpenAPI spec, to describe the desired API for v2, to be functional after the completion of all v2 issues.

#### Do you have any concerns with this PR?
Using basic auth on network integrations assumes that all networks support providing auth headers.  If not, can pivot to put apiKey in the url.

#### How can the reviewer verify this PR?
```
npm install
npm run openapi:serve
```
Then view localhost:5000 to see API docs

#### Any background context you want to provide?
No

#### Screenshots or logs (if appropriate)

#### Questions:
- Have you connected this PR to the issue it resolves? #333 
- Does the documentation need an update? Included
- Does this add new dependencies? No
- Have you added unit or functional tests for this PR? No
